Project-3/psaurav1290/README.md

```diff
@@ -0,0 +1,48 @@
+# README
```

Comment:
Please add a link to your website and optionally screenshots of the functionality here itself.

Thanks!

Reply:
Link to the git page - [**WeatherApp**](https://psaurav1290.github.io/weatherapp/) 

1. On opening, a **prompt is generated to grant location access** so that the page can be customized on the basis of current location. If permission not granted within 10 seconds, the location is set to default.
![Page customized on current weather condition](https://user-images.githubusercontent.com/60401832/85955894-7578a180-b99f-11ea-8c9c-e0662d6db53f.png)

2. The page shows **forecast for next 5 days** and **weather records of past 5 days** using weather icons.
![Forecast and past records](https://user-images.githubusercontent.com/60401832/85955988-f9cb2480-b99f-11ea-8c30-853fd59f8e59.png)

3. **Clicking each of them reveals the details** like weather condition, description, temperatuere, windspeed and humidity.
![Clickable icons when clicked reveals details](https://user-images.githubusercontent.com/60401832/85956027-36971b80-b9a0-11ea-8def-6619132fb323.png)

4. Since a large part of traffic using weather apps are mobile users, so site is mobile friendly and well **optimized for small screens**.
![Mobile-friendly site](https://user-images.githubusercontent.com/60401832/85956037-4adb1880-b9a0-11ea-9cf5-caeaeab1b37c.png)

5. **Custom scrolling effects, well-built colour schemes, clear graphics, auto-hide header & favicon** is added to enhance overall user experience.

6. **Errors and warning are displayed** as alerts and all exceptions are handled.

- *Warnings*
![Warnings are displayed in alert boxes](https://user-images.githubusercontent.com/60401832/85956334-c938ba00-b9a2-11ea-8c79-f6a867317b9c.png)
- *Errors*
![Errors are displayed as alert boxes](https://user-images.githubusercontent.com/60401832/85956336-cb027d80-b9a2-11ea-8d44-0dd90ff9fa68.png)